### PR TITLE
#88 Implement `pazuzu build` for Git storage

### PR DESCRIPTION
**Pull request by netme**
Issue #88 

📙 

⚠️ Breaks existing API integration. API integration still have to be rework in #98 .

⚠️ Removes the test for `test_spec.json` to be reworked in #96

### CODE REVIEW

File: cli/pazuzu/commands.go [hunk 1]
```diff
@@ -137,7 +137,39 @@ var buildCmd = cli.Command{
 
 // Fetches and builds features into a docker image.
 func buildFeatures(c *cli.Context) error {
-	return ErrNotImplemented
+	// TODO: Make file configurable via CLI args
```
Comment (pgronkiewicz) [inline cli/pazuzu/commands.go]: Please reference Github issues every time a TODO entry is created :)

File: dockerfilewriter.go [hunk 2]
```diff
@@ -3,8 +3,11 @@ package pazuzu
 import (
 	"bytes"
 	"fmt"
-	"github.com/docker/docker/builder/dockerfile/parser"
 	"strings"
+
+	"github.com/docker/docker/builder/dockerfile/parser"
+
+	"github.com/zalando-incubator/pazuzu/storageconnector"
```
Comment (pgronkiewicz) [inline dockerfilewriter.go]: So it will stop working once we move to main zalando org, right?

File: cli/pazuzu/commands.go (hunk 1 above)
Comment (netme) [inline cli/pazuzu/commands.go]: Fixed

**Comment by pgronkiewicz** [comment]
:+1:

**Comment by netme** [comment]
👍

**Comment by netme** [comment]
Reapproved after rebase

**Pull request merged**